src/components/loaders/inline-loader.component.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styles from './inline-loader.scss';
+import { InlineLoading } from '@carbon/react';
+
+const InlineLoader: React.FC = () => (
+  <div className={styles.formField}>
+    <span className="cds--label"></span>
```

Comment:
Looks like an empty span

Reply:
That was me trying to align the loader. I think it can be improved.

Reply:
That was me trying to align the loader. I think it can be improved.

Reply:
Can we use a flex grid or something?